.eslintrc.json

```diff
@@ -1,3 +1,47 @@
 {
-  "extends": ["next/core-web-vitals", "prettier"]
+  "extends": ["next/core-web-vitals", "prettier"],
+  "parserOptions": {
+    "sourceType": "module"
+  },
+  "env": { "es6": true },
+  "plugins": ["simple-import-sort"],
+  "rules": {
```

Comment:
Could also add to the rules:
```json
"import/first": "error",
"import/newline-after-import": "error",
"import/no-duplicates": "error"
```
Need to double-check if these rules are already present in NextJS